lt/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="lt">
+<head>
+  <title>Ne klauskite, kad klausti, klauskite iš karto</title>
+  <meta property="og:title" content="Ne klauskite, kad klausti, klauskite iš karto" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ru/" />
+  <meta property="og:locale" content="lt_LT" />
+  <meta property="og:image" content="https://dontasktoask.com/favicon.png"/>
+  <link rel="icon" href="/favicon.png" type="image/png" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Ne klauskite, kad klausti, klauskite iš karto</h1>
+    <p>
+	  Pokalbiuose, kuriuose aš būnu, kartais pasitaiko, kad kažkas klausia
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Kažkas iš čia rašo ant Java kalbos?
+      </p>
+    </blockquote>
+    <p>
+      Tai yra blogas klausimų formatas dėl daugelių priežasčių. <em>Tiesą sakant</em>
+      Žmogus klausia:
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+		Kažkas čia rašo "Java" kalba ir nori skirti laiko mano analizei problemos,
+		kad ir kuo tai baigtųsi,
+		net jei tai yra tikrai nėra labai susijęs su Java ir bet kas,
+		neturintis žinių apie Java ar galėtumėte atsakyti į mano klausimą?
+      </p>
+    </blockquote>
+    <p>
+		Yra daug priežasčių,kodėl žmonės, turintys žinių tinkamoje srityje,
+		gali neatsakyti.Tokiu būdu užduodami klausimą,jūs klausiate apie daugiau,nei
+		nei atrodo iš pirmo žvilgsnio.
+    </p>
+    <p>
+	Jūs klausiate žmonių, ar jie nori prisiimti atsakomybę. Tu abejoja jų pasitikėjimu savo įgūdžiais.
+	Taip pat žr.: nepagrįstai atsisakyti kitų žmonių pagalbos. Pavyzdžiui, aš dažnai atsakyti į klausimus,
+	susijusius su kalbomis ar bibliotekomis, kurias aš niekada nenaudojamas,
+	nes atsakymai yra tik sveikas protas (programuotojo požiūriu).

```

Comment:
another place where ur mixing 'klausiate' and 'prašote', which shifts the meaning of the sentence

plus ur using incorrect pronouns and gendering ( e.g. Tu abejoja => Tu abejoji )

this sentence should b re-translated, idk if theres a proper word for libraries but my old informatics teacher used to call it 'biblioteka' so idk, feel free to find a better word :

> Jūs prašote žmonių prisiimti atsakomybę. Jūs keliate abejonių dėl žmonių gebėjimų. Jūs taip pat nereikalingai atskiriate kitus žmones. Dažnai atsakau į klausimus, susijusius su programavimo kalbomis ar bibliotekomis, kurių niekada nenaudojau, nes atsakymai (programuotojo požiūriu) yra visiems žinomi.

kinda a botched translation bc its hard to translate this properly